### ISSUE TITLE
There's no spacing between words

### ISSUE THREAD

**Issue by titan901** [akaalias/obsidian-extract-pdf-highlights#5]
After extracting the highlighted pdf, the extracted .md file has no spacing between words

**Comment by maoyimin** [comment]
i have the same problem

**Comment by allisondunning** [comment]
Just adding that I have the same problem. The PDF I used is a journal article in 2 column format.

**Comment by lkhphuc** [comment]
Same for me. Tried a few different ones from my Zotero and same result.

**Comment by legend053** [comment]
Same here. No space between words. This plugin is really helpful for students and educators.

**Comment by ChaliZhg** [comment]
Same here. I'm attaching my system's specification and a pdf file in case it helps anything.
OS: Ubuntu 18.04
Obsidian: 0.11.09
PDF Highlights: 0.0.4
PDF file: [dam2019.pdf](https://github.com/akaalias/obsidian-extract-pdf-highlights/files/6244523/dam2019.pdf) with a single annotation on page 1 made with Okular; btw, Zotfile could capture this annotation correctly.
Screenshot: ![Screenshot_pdf](https://user-images.githubusercontent.com/6869975/113315820-f9d0ca00-930d-11eb-8fb0-ffa21ed75afd.png)

**Comment by Galacticminingoperation** [comment]
Same here.

**Comment by Viishied** [comment]
Anyone know of a solution, or is it a no go until there's a fix?

**Comment by ndharari** [comment]
same problem here also

**Comment by owldown** [comment]
This is true even in the demo video/gif. Really limits the usefulness of this plugin.

**Comment by codeRSH** [comment]
For me, the space between words gets removed when the words are on different lines. Can this be fixed. Great plugin otherwise!
